### ISSUE TITLE
links under "improve this page" are broken

### ISSUE THREAD

**Issue by cengel** [datacarpentry/r-socialsci#232]
It appears that the files were moved from episode to episode_rmd, so the links under "Improve this page" in the top menu bar of the materials need to change as well. For example:

change
`https://github.com/datacarpentry/r-socialsci/edit/master/_episodes/02-starting-with-data.md`

to
`https://github.com/datacarpentry/r-socialsci/edit/master/_episodes_rmd/02-starting-with-data.Rmd`

**Comment by kelseygonzalez** [comment]
Hi @cengel ,
These links are not actually broken: the /_episodes_rmd/ files get converted into .md files in the website building process. We want people to update the .rmd files, though, as these are the source files! 

Thanks for having a keen eye and pointing this out. Your issues are helping us improve the lesson!

**Comment by cengel** [comment]
Hi @kelseygonzalez,
Apologies, I did confuse `md` and `rmd` in my earlier message, but some of the links still seem broken.

Right now if I go to the following pages and click on 'Improve this page" I get directed to the .md file.
https://datacarpentry.org/r-socialsci/
https://datacarpentry.org/r-socialsci/setup.html
https://datacarpentry.org/r-socialsci/00-intro/index.html
https://datacarpentry.org/r-socialsci/01-intro-to-r/index.html

If I go to the following pages and click on 'Improve this page" I get a 404 error.
https://datacarpentry.org/r-socialsci/02-starting-with-data/index.html
https://datacarpentry.org/r-socialsci/03-dplyr-tidyr/index.html
https://datacarpentry.org/r-socialsci/04-ggplot2/index.html 
https://datacarpentry.org/r-socialsci/0x-json/index.html

**Comment by fmichonneau** [comment]
Thanks for noticing this @cengel! I submitted #240 which should address the 404 you identified.

**Comment by dafnevk** [comment]
I still get a similar issue, if I click 'improve this page' e.g. from the lesson website, e. from episode 3, I get redirected to https://datacarpentry.org/edit//_episodes_rmd/03-dplyr-tidyr.Rmd which is a broken link.

**Comment by juanfung** [comment]
@dafnevk Thank you for noticing that the issue persists! We will investigate and get it fixed asap.

**Comment by juanfung** [comment]
@fmichonneau I looked at `navbar.html` and I thought the problem was the `repo_url` variable, but in `gh_variables.html` it looks like `repo_url` is set correctly. However, when a user clicks on "Improve this page," the `repo_url` appears as `datacarpentry.org`. Also note that `default_branch` variable should be `master` but no branch appears in the link. 

In short, I have no idea how to fix! :)

**Comment by fmichonneau** [comment]
Thank you for bringing this up. This was caused by the new approach to build the site using GitHub Actions. The approach was building the html files for the site and was pushing this directly to the `gh-pages` branch. The website rendered correctly but it meant that the logic used to get the repository info we need from the GitHub API (name of the repository, default branch, ...) was not used. In turn, this was breaking the "Improve this page" links. I switched the GitHub Action to only render the Rmd files to markdown and copy the entire content of the repository to the `gh-pages` branch and from there GitHub pages will build the Jekyll site which should make the links functional again.
